Project.toml

```
@@ -1,32 +1,34 @@
-name = "Sundials"
-uuid = "c3572dad-4567-51f8-b174-8c6c989267f4"
-authors = ["Chris Rackauckas <accounts@chrisrackauckas.com>"]
-version = "v3.8.2"
-
-[deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
-Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
-DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
-SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
-SuiteSparse = "4607b0f0-06f3-5cda-b6b1-a6196a1729e9"
-LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
-Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
-
-[compat]
-julia = "1"
-DiffEqBase = "6"
-DataStructures = "0.17.0"
-BinaryProvider = "0.5"
-Reexport = "0.2"
-
-[extras]
-DiffEqProblemLibrary = "a077e3f3-b75c-5d7f-a0c6-6bc4c8ec64a9"
-ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
-Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
-DiffEqOperators = "9fdde737-9c7f-55bf-ade8-46b3f136cc48"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-
-[targets]
-test = ["Test", "DiffEqProblemLibrary", "ForwardDiff", "Compat", "DiffEqOperators"]
+name = "Sundials"
+uuid = "c3572dad-4567-51f8-b174-8c6c989267f4"
+authors = ["Chris Rackauckas <accounts@chrisrackauckas.com>"]
+version = "3.8.2"
```

```suggestion
version = "3.8.3"
```
We need a new patch release.